pkg/commands/drift/github.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package drift
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/abcxyz/guardian/pkg/github"
+)
+
+const (
+	issueTitle = "IAM drift detected"
+	issueBody  = `We've detected a drift between your submitted IAM policies and actual
+		IAM policies.
```

Comment:
Nit: these appear to be indented with tabs, which _will_ render in the GitHub PR.

Reply:
Done.